applications/RomApplication/tests/structural_dynamic_test_files/GiD_Model.mdpa

```diff
@@ -0,0 +1,496 @@
+Begin ModelPartData
```

Comment:
the file name is not very representative....

Reply:
Names were changes